week-6-benchmarking/README.md

```diff
@@ -0,0 +1,171 @@
+# Benchmarking
+
+## PR benchmark for your model forward pass
+
+Run:
+
+```bash
+make benchmark-forwardpass
+```
+
+Results:
+
+```
+---------------- benchmark: 2 tests ----------------
+Name (time in ms)                     Mean          
+----------------------------------------------------
+test_benchmark_forwardpass_gpu      6.8554 (1.0)    
+test_benchmark_forwardpass_cpu     35.3109 (5.15)   
+----------------------------------------------------
+```
+
+## PR benchmark for your model API
+
+1. Build docker image with the API
+
+```bash
+cd api/
+docker build --rm -t bert-api .
+docker tag bert-api:latest yevhenk10s/bert-api:latest
+docker push yevhenk10s/bert-api:latest
+```
+
+2. Test docker image locally
+
+```bash
+docker run -p 5000:5000 yevhenk10s/bert-api:latest
+curl localhost:5000
+curl -X POST localhost:5000 -H 'Content-Type: application/json' -d '{"texts": ["hello, world", "this is a sentance"]}'
+```
+
+3. Run test benchmark
+
+```bash
+make benchmark-api-local
+```
+
+Results:
+
+
+| Type | Name       | # reqs | # fails   | Avg  | Min | Max   | Med  | req/s  | failures/s |
+| ---- | ---------- | ------ | --------- | ---- | --- | ----- | ---- | ------ | ---------- |
+| POST | /          | 2200   | 80(3.64%) | 2988 | 424 | 14541 | 2000 | 108.40 | 1.20       |
+|      | Aggregated | 2200   | 80(3.64%) | 2988 | 424 | 14541 | 2000 | 108.40 | 1.20       |
```

Comment:
same, what was the env: machine type, CPU, RAM, noisy neighbours, etc

Reply:
Readme updated